var/spack/repos/builtin/packages/py-numpy/package.py

```diff
@@ -310,20 +312,32 @@ def write_library_dirs(f, dirs):
                         )
                     )
 
+            if '^cray-libsci' in spec:
```

Comment:
Since `fujitsu-ssl2` and `cray-libsci` both fall under the generic "no special support so just use [blas] and [lapack]" I wonder if they can be combined into a single if-statement like we do for other clusters (e.g. MKL).

Reply:
Looks like it wasn't necessary to use `libs.names[0]` to get this to work for Cray, so I have combined the `cray-libsci` and `netlib-lapack` config. It looks like `fujitsu-ssl2` needs some additional link args, I'm not in a position to test the `fujitus-ssl2` libraries, so I'll leave that block as it is.